### PR TITLE
SpdxLicenseAliasMapping: Add APACHEv2

### PR DESCRIPTION
**Pull request by tsteenbe**
Found in
https://git.coolaj86.com/coolaj86/semver-utils.js/src/tag/v1.1.4/package.json#L21.

Signed-off-by: Thomas Steenbergen <thomas.steenbergen@here.com>

### CODE REVIEW

File: spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt [hunk 1]
```diff
@@ -40,6 +40,7 @@ object SpdxDeclaredLicenseMapping {
         "3-Clause BSD License" to BSD_3_CLAUSE.toExpression(),
         "3-clause bdsl" to BSD_3_CLAUSE.toExpression(),
         "Affero General Public License (AGPL) v. 3" to AGPL_3_0_ONLY.toExpression(),
+        "APACHEv2" to APACHE_2_0.toExpression(),
```
Comment (fviernau) [inline spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt:43]: Does it need to go one line further down?

Comment (sschuberth) [reply spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: It anyway needs to go to `SpdxLicenseAliasMapping`.

Comment (tsteenbe) [reply spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: Thx @sschuberth  for spotting that one, updated my PR to make the change to `SpdxLicenseAliasMapping`.

File: spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt [hunk 2]
```diff
@@ -41,6 +41,7 @@ object SpdxLicenseAliasMapping {
         "AFLv2.1" to AFL_2_1,
         "agpl" to AGPL_3_0_ONLY,
         "ALv2" to APACHE_2_0,
+        "APACHEv2" to APACHE_2_0,
```
Comment (fviernau) [inline spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt:44]: Should it go above line 49?

Comment (sschuberth) [reply spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt:44]: I'll sort the whole list as part of another PR.

File: spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt (hunk 1 above)
Comment (sschuberth) [reply spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: See https://github.com/oss-review-toolkit/ort/pull/2460.

**Pull request merged**